src/codecoverage/bot/tests/test_artifacts.py

```diff
@@ -27,6 +27,14 @@ def FAKE_ARTIFACTS_DIR(tmpdir):
     return tmpdir.strpath
 
 
+def test_generate_path(FAKE_ARTIFACTS_DIR):
+    a = ArtifactsHandler([], parent_dir=FAKE_ARTIFACTS_DIR)
+    artifact_jsvm = {'name': 'code-coverage-jsvm.info'}
+    artifact_grcov = {'name': 'code-coverage-grcov.zip'}  
```

Comment:
There are some trailing whitespaces here, which make the build fail. Could you remove them?

Reply:
Yep, booting my VM now. Sorry about that!

Reply:
Had to fetch some changes so it asked me to merge again - hopefully that builds properly now.